### ISSUE TITLE
Source assets tab is laggy

### ISSUE THREAD

**Issue by vertxxyz** [Unity-Technologies/BuildReportInspector#15]
With this tab opening onto a page that is drawing thousands of included assets it's unusable in my project.
Editing the script to land on the collapsed importer type view makes it vaguely usable, but it seems this isn't very scalable.
This should use a list view functionality where it's only actually displaying the few visible elements.

**Comment by xjjon** [comment]
Same problem here, it is pretty much unusable as Unity becomes unresponsive. I can just take a screenshot of the top size assets and then that's about it.

**Comment by NickDeLaRosaLion** [comment]
Same issue here. Makes it unusable

**Comment by yanghu28** [comment]
卡蒙了，

**Comment by yanghu28** [comment]
说实话不需要缩略图的。只需要显示名字分好类，然后再好一点就是点击可以看到索引 的内容，不需要把视图显示出来

**Comment by SkowronskiAndrew** [comment]
I agree this has been quite unusable for large numbers of objects, e.g. for large builds.

#39 will fix the SourceAsset tab a lot by collapsing redundant entries together. And if the number of entries it is large it simply won't display so that other parts of the build report UI can be used.

Beyond that the build report still gets quite large so there can be some performance problems but i will close this ticket when that PR lands and we can address further bottlenecks if new issues are opened.
